pwndbg/lib/cache.py

```diff
@@ -132,7 +145,7 @@ def inner(func: Callable[P, T]) -> Callable[P, T]:
 
         @wraps(func)
         def decorator(*a: P.args, **kw: P.kwargs) -> T:
-            if IS_CACHING:
+            if IS_CACHING and not any((IS_CACHING_DISABLED_FOR[e] for e in event_names)):
```

Comment:
We really don't want this check here; if we need this ability on the long run, this should be changed somehow

Reply:
I wonder where else we could put it, or what other way we could achieve this, since this is fairly important, seeing as without it, caches like `thread` on LLDB would be the same as `forever`.

Reply:
I mean... probably change the impl for those that should just call the result instead of having an additional if condition here?